### PR TITLE
Create Releases “2023-01-09-december-2022”

### PR DESCRIPTION
**Pull request by amanifarooque**
Automatically generated by Netlify CMS

### CODE REVIEW

File: _releases/2023-01-09-december-2022.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+filename-custom: december-2022
+last_modified_at: January 9, 2023
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements 
+* **ElasticSearch Upgrade**: We've completed the upgrade to our ElasticSearch version across our system.
+* **Rails Upgrade**: We've completed the upgrade to our Rails version in our main application.
+* **SERP Redesign**: We've installed the necessary components to support React in our Rails application.
```
Comment (dawnpm) [inline _releases/2023-01-09-december-2022.md:14]: Let's say "Results Page Redesign"

File: _releases/2023-01-09-december-2022.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* **Indexing Improvements**: We've added the ability for any member of our team to queue a specific set of URLs for reindexing, rather than the entire domain.
```
Comment (dawnpm) [inline _releases/2023-01-09-december-2022.md:15]: how about "to enqueue for reindexing a specific set of URLs, rather than the entire domain"

Comment (amanifarooque) [reply _releases/2023-01-09-december-2022.md]: updated!

File: _releases/2023-01-09-december-2022.md (hunk 1 above)
Comment (amanifarooque) [reply _releases/2023-01-09-december-2022.md]: updated!

**Review by MyNameIsMissing — APPROVED** [review]
LGTM assuming the 2022 release year is correct since this is intended to document December 2022 changes.

**Comment by amanifarooque** [comment]
@dawnpm Bumping this PR!

**Review by dawnpm — APPROVED** [review]

**Pull request merged**